.drone.yml

```diff
@@ -210,12 +212,19 @@ services:
       matrix:
         NEED_SERVER: true
 
-  selenium:
+  chrome:
     image: selenium/standalone-chrome-debug:latest
     pull: true
     when:
       matrix:
-        TEST_SUITE: acceptance
+        BROWSER: chrome
+
+  firefox:
+    image: selenium/standalone-firefox-debug:3.141.59-mercury
```

Comment:
why not `:latest`?

Reply:
Oh, I used for testing purpose. I should keep the `latest` one there.